extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmReflections.java

```diff
@@ -102,6 +102,17 @@ public void registerCoreReflections(BuildProducer<ReflectiveClassBuildItem> refl
         simpleConstructor(reflectiveClass, "org.hibernate.internal.CoreMessageLogger_$logger");
     }
 
+    @BuildStep
+    public void registerEnversReflections(BuildProducer<ReflectiveClassBuildItem> reflectiveClass) {
+        reflectiveClass.produce(
+                new ReflectiveClassBuildItem(false, false, "org.hibernate.tuple.component.DynamicMapComponentTuplizer"));
+        reflectiveClass.produce(new ReflectiveClassBuildItem(true, false, "org.hibernate.envers.DefaultRevisionEntity"));
+        reflectiveClass.produce(
+                new ReflectiveClassBuildItem(false, false, "org.hibernate.persister.entity.SingleTableEntityPersister"));
+        reflectiveClass
+                .produce(new ReflectiveClassBuildItem(false, false, "org.hibernate.tuple.entity.DynamicMapEntityTuplizer"));
```

Comment:
Could you register the two with `org.hibernate.tuple` with the others above (at the beginning of the above method)?

We would only have the Envers-specific class here.